modules/opensimplex/SCsub

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env python
+
+Import('env')
+
+thirdparty_dir = '#thirdparty/misc/'
+thirdparty_source = thirdparty_dir + 'open-simplex-noise.c'
+env.Append(CPPPATH=thirdparty_dir)
```

Comment:
Since it's only one file, and not likely to be packaged by Linux distros and thus unbundled for the time being, I'd suggest not to include `thirdparty/misc` to the include path, and instead include `thirdparty/misc/open-simplex-noise.h` directly in the relevant files.

This saves us an extra 18 characters on every single compilation command line, which is nice to avoid logs growing to crazy sizes and Windows aborting the build due to too long instructions.